### ISSUE TITLE
v3.1.3: IndexError: list index out of range

### ISSUE THREAD

**Issue by GerdRuetten** [gcobb321/icloud3_v3#372]
Hi @gcobb321 

i see this two error first time in the HA-Log

```
Dieser Fehler stammt von einer benutzerdefinierten Integration

Logger: custom_components.icloud3
Quelle: custom_components/icloud3/support/event_log.py:613
Integration: iCloud3 v3 (Dokumentation, Probleme)
Erstmals aufgetreten: 14:30:38 (1 Vorkommnisse)
Zuletzt protokolliert: 14:30:38

list index out of range
Traceback (most recent call last):
  File "/config/custom_components/icloud3/support/event_log.py", line 613, in _shrink_event_recds
    elr_recd     = self.event_recds[x]
                   ~~~~~~~~~~~~~~~~^^^
IndexError: list index out of range
```
AND

```
Dieser Fehler stammt von einer benutzerdefinierten Integration

Logger: custom_components.icloud3
Quelle: custom_components/icloud3/support/event_log.py:625
Integration: iCloud3 v3 (Dokumentation, Probleme)
Erstmals aufgetreten: 17:07:39 (1 Vorkommnisse)
Zuletzt protokolliert: 17:07:39

list assignment index out of range
Traceback (most recent call last):
  File "/config/custom_components/icloud3/support/event_log.py", line 625, in _shrink_event_recds
    del self.event_recds[x]
        ~~~~~~~~~~~~~~~~^^^
IndexError: list assignment index out of range
```

**Comment by gcobb321** [comment]
The _Event Log_ is deleting old records and ran into something when it was doing that. It should not have an effect on anything.

**Comment by GerdRuetten** [comment]
Yes, I can confirm that. The integration runs without any problems. I only saw the error in the HA log by chance

**Comment by gcobb321** [comment]
This is fixed in v3.1.4
